catalog/imaging/imaging-transformer-nitf/src/test/java/org/codice/alliance/transformer/nitf/image/PreStoragePluginTest.java

```diff
@@ -78,7 +80,12 @@ public void setUp()
         when(updateStorageRequest.getContentItems()).thenReturn(contentItems);
         when(contentItem.getMetacard()).thenReturn(metacard);
         when(contentItem.getId()).thenReturn("101ABC");
-        when(contentItem.getInputStream()).thenReturn(getInputStream(GEO_NITF));
+        when(contentItem.getInputStream()).thenAnswer(new Answer<InputStream>() {
```

Comment:
You can't do `when(contentItem.getInputStream()).thenAnswer(invocationOnMock -> getInputStream(GEO_NITF));`?